src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -439,16 +510,19 @@ export default class PhotoField extends React.Component {
           {!isDone && this.state.src && <div className="cropper-container-outer">
             <Cropper
               ref="cropper"
+              key={smallScreen ? 'smallScreen' : 'largeScreen'}
               ready={this.setCropBox}
               responsive
               src={this.state.src}
               aspectRatio={1}
               cropBoxMovable={false}
               cropend={this.handleCropend}
+              cropmove={this.handleCropMove}
+              minContainerHeight={smallScreen ? 240 : 300}
               toggleDragModeOnDblclick={false}
               dragMode="move"
               guides={false}
-              viewMode={1}
+              viewMode={0}
```

Comment:
The old `viewMode` value was 1 which
> `restrict the crop box to not exceed the size of the canvas.`

The side effect is that the cropper size is adjusted to meet the default behavior of displaying the full image when it is initially loaded and the cropper size cannot be manually set. 

`viewMode={0}` allows the cropBox size to be manually set but does not constrain the canvas movement to the bounds of the cropBox necessitating handling this manually. 
<img width="588" alt="screen shot 2018-02-06 at 20 38 51" src="https://user-images.githubusercontent.com/4998130/35895812-bdcfa310-0b7e-11e8-8f37-4faa405d4e43.png">

Reply:
This seems to allow users to adjust the size of the crop box, which is something we wanted to prevent.

Reply:
Good catch. I just made a change that sets `cropBoxResizable` to false. It removes the grey border- which I guess was there to represent a draggable element.
<img width="550" alt="screen shot 2018-02-07 at 08 40 14" src="https://user-images.githubusercontent.com/4998130/35922219-9efb45f2-0be2-11e8-9572-92ac74c70313.png">